packages/blocks/src/rating-scale-answer/index.js

```diff
@@ -42,8 +41,10 @@ const RatingScaleAnswer = ( { attributes, className } ) => {
 		<ButtonAnswer
 			attributes={ attributes }
 			className={ classes }
-			inputProps={ inputProps }
 			isMultiSelect={ false }
+			isSelected={ isSelected }
+			onChange={ ( event ) => onUpdate( event.target.value, isSelected ) }
```

Comment:
The `event -> value` mapping should be done in our input components. So here it's just `onChange={ onUpdate }` or `onChange={ setFieldValue }`.